ansible/admin_users.yml

```
@@ -5,6 +5,7 @@
 # See the `-k` and `-K` flags when running ansible-playbook.
 #
 - hosts: all
+  environment: "{{ proxy_env }}"
```

Hi, could you change all these lines to

```suggestion
  environment: "{{ proxy_env | default({}) }}"
```
? This way, the playbook can be run as before for those people who do not need any proxy environment variables without any changes.

Otherwise, this looks good, thanks!